packages/config/src/configDefaults.ts

```diff
@@ -24,10 +32,10 @@ const _values = ({ truffleDirectory, workingDirectory, network }) => {
     resolver: null,
     artifactor: null,
     ethpm: {
-      ipfs_host: "ipfs.infura.io",
-      ipfs_protocol: "https",
-      registry: "0x8011df4830b4f696cd81393997e5371b93338878",
-      install_provider_uri: "https://ropsten.infura.io/truffle"
+      ipfs_host: 'ipfs.infura.io',
```

Comment:
Also, prettier changes strings to use double quotes and not single quotes wherever possible.

Reply:
Resolved